### ISSUE TITLE
Rails new command very verbose 

### ISSUE THREAD

**Issue by faraazahmad** [rails/rails#50237]
### Steps to reproduce

```sh
rails new <appname>
```

### Expected behavior

I do recognize this issue might be trivial so feel free to close this. 

For someone familiar with Rails, they would see all the files/folders being created and gems being installed and they know whats going on. For someone who's totally new (who is someone the Rails community is trying to be more welcoming to) this might be intimidating, I've been using Yarn recently and I think it prevents the information overload pretty well. 

By default, I would expect the output to be minimal, for example, see the output of creating a new Nuxtjs app. The current output can still be viewed using a `verbose` flag. 

### Actual behavior

I get this giant wall of text when creating a new rails app:

```
      create  
      create  README.md
      create  Rakefile
      create  .ruby-version
      create  config.ru
      create  .gitignore
      create  .gitattributes
      create  Gemfile
         run  git init from "."
Initialized empty Git repository in /Users/faraaz/oss/reseed/.git/
      create  app
      create  app/assets/config/manifest.js
      create  app/assets/stylesheets/application.css
      create  app/channels/application_cable/channel.rb
      create  app/channels/application_cable/connection.rb
      create  app/controllers/application_controller.rb
      create  app/helpers/application_helper.rb
      create  app/jobs/application_job.rb
      create  app/mailers/application_mailer.rb
      create  app/models/application_record.rb
      create  app/views/layouts/application.html.erb
      create  app/views/layouts/mailer.html.erb
      create  app/views/layouts/mailer.text.erb
      create  app/assets/images
      create  app/assets/images/.keep
      create  app/controllers/concerns/.keep
      create  app/models/concerns/.keep
      create  bin
      create  bin/rails
      create  bin/rake
      create  bin/setup
      create  Dockerfile
      create  .dockerignore
      create  bin/docker-entrypoint
      create  config
      create  config/routes.rb
      create  config/application.rb
      create  config/environment.rb
      create  config/cable.yml
      create  config/puma.rb
      create  config/storage.yml
      create  config/environments
      create  config/environments/development.rb
      create  config/environments/production.rb
      create  config/environments/test.rb
      create  config/initializers
      create  config/initializers/assets.rb
      create  config/initializers/content_security_policy.rb
      create  config/initializers/cors.rb
      create  config/initializers/filter_parameter_logging.rb
      create  config/initializers/inflections.rb
      create  config/initializers/new_framework_defaults_7_1.rb
      create  config/initializers/permissions_policy.rb
      create  config/locales
      create  config/locales/en.yml
      create  config/master.key
      append  .gitignore
      create  config/boot.rb
      create  config/database.yml
      create  db
      create  db/seeds.rb
      create  lib
      create  lib/tasks
      create  lib/tasks/.keep
      create  lib/assets
      create  lib/assets/.keep
      create  log
      create  log/.keep
      create  public
      create  public/404.html
      create  public/422.html
      create  public/500.html
      create  public/apple-touch-icon-precomposed.png
      create  public/apple-touch-icon.png
      create  public/favicon.ico
      create  public/robots.txt
      create  tmp
      create  tmp/.keep
      create  tmp/pids
      create  tmp/pids/.keep
      create  tmp/cache
      create  tmp/cache/assets
      create  vendor
      create  vendor/.keep
      create  test/fixtures/files
      create  test/fixtures/files/.keep
      create  test/controllers
      create  test/controllers/.keep
      create  test/mailers
      create  test/mailers/.keep
      create  test/models
      create  test/models/.keep
      create  test/helpers
      create  test/helpers/.keep
      create  test/integration
      create  test/integration/.keep
      create  test/channels/application_cable/connection_test.rb
      create  test/test_helper.rb
      create  test/system
      create  test/system/.keep
      create  test/application_system_test_case.rb
      create  storage
      create  storage/.keep
      create  tmp/storage
      create  tmp/storage/.keep
      remove  config/initializers/cors.rb
      remove  config/initializers/new_framework_defaults_7_1.rb
         run  bundle install
Fetching gem metadata from https://rubygems.org/...........
Resolving dependencies...
Fetching rake 13.1.0
Installing rake 13.1.0
Fetching mutex_m 0.2.0
Fetching base64 0.2.0
Fetching bigdecimal 3.1.4
Fetching stringio 3.1.0
Fetching date 3.3.4
Fetching public_suffix 5.0.4
Fetching bindex 0.8.1
Fetching msgpack 1.7.2
Installing mutex_m 0.2.0
Fetching rubyzip 2.3.2
Installing base64 0.2.0
Installing stringio 3.1.0 with native extensions
Installing bindex 0.8.1 with native extensions
Fetching websocket 1.2.10
Installing bigdecimal 3.1.4 with native extensions
Installing msgpack 1.7.2 with native extensions
Installing date 3.3.4 with native extensions
Installing public_suffix 5.0.4
Installing rubyzip 2.3.2
Installing websocket 1.2.10
Fetching sqlite3 1.6.9 (x86_64-darwin)
Fetching drb 2.2.0
Fetching nokogiri 1.15.5 (x86_64-darwin)
Installing drb 2.2.0
Fetching net-protocol 0.2.2
Fetching sprockets 4.2.1
Installing net-protocol 0.2.2
Fetching puma 6.4.0
Installing sprockets 4.2.1
Fetching reline 0.4.1
Fetching addressable 2.8.5
Installing puma 6.4.0 with native extensions
Installing reline 0.4.1
Fetching selenium-webdriver 4.15.0
Installing addressable 2.8.5
Fetching net-smtp 0.4.0
Installing net-smtp 0.4.0
Installing sqlite3 1.6.9 (x86_64-darwin)
Installing selenium-webdriver 4.15.0
Installing nokogiri 1.15.5 (x86_64-darwin)
Fetching xpath 3.2.0
Installing xpath 3.2.0
Fetching capybara 3.39.2
Installing capybara 3.39.2
Fetching bootsnap 1.17.0
Fetching net-imap 0.4.7
Installing bootsnap 1.17.0 with native extensions
Installing net-imap 0.4.7
Fetching jbuilder 2.11.5
Fetching sprockets-rails 3.4.2
Fetching importmap-rails 1.2.3
Fetching stimulus-rails 1.3.0
Fetching turbo-rails 1.5.0
Fetching web-console 4.2.1
Installing sprockets-rails 3.4.2
Installing jbuilder 2.11.5
Installing web-console 4.2.1
Installing stimulus-rails 1.3.0
Installing importmap-rails 1.2.3
Installing turbo-rails 1.5.0
Bundle complete! 14 Gemfile dependencies, 82 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
         run  bundle lock --add-platform=x86_64-linux
Fetching gem metadata from https://rubygems.org/.........
Resolving dependencies...
Writing lockfile to /Users/faraaz/oss/reseed/Gemfile.lock
         run  bundle binstubs bundler
       rails  importmap:install
       apply  /Users/faraaz/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/importmap-rails-1.2.3/lib/install/install.rb
  Add Importmap include tags in application layout
      insert    app/views/layouts/application.html.erb
  Create application.js module as entrypoint
      create    app/javascript/application.js
  Use vendor/javascript for downloaded pins
      create    vendor/javascript
      create    vendor/javascript/.keep
  Ensure JavaScript files are in the Sprocket manifest
      append    app/assets/config/manifest.js
  Configure importmap paths in config/importmap.rb
      create    config/importmap.rb
  Copying binstub
      create    bin/importmap
         run  bundle install
Bundle complete! 14 Gemfile dependencies, 82 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
         run  bundle lock --add-platform=x86_64-linux
Writing lockfile to /Users/faraaz/oss/reseed/Gemfile.lock
       rails  turbo:install stimulus:install
       apply  /Users/faraaz/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/turbo-rails-1.5.0/lib/install/turbo_with_importmap.rb
  Import Turbo
      append    app/javascript/application.js
  Pin Turbo
      append    config/importmap.rb
         run  bundle install
Bundle complete! 14 Gemfile dependencies, 82 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
         run  bundle lock --add-platform=x86_64-linux
Writing lockfile to /Users/faraaz/oss/reseed/Gemfile.lock
Run turbo:install:redis to switch on Redis and use it in development for turbo streams
       apply  /Users/faraaz/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/stimulus-rails-1.3.0/lib/install/stimulus_with_importmap.rb
  Create controllers directory
      create    app/javascript/controllers
      create    app/javascript/controllers/index.js
      create    app/javascript/controllers/application.js
      create    app/javascript/controllers/hello_controller.js
  Import Stimulus controllers
      append    app/javascript/application.js
  Pin Stimulus
  Appending: pin "@hotwired/stimulus", to: "stimulus.min.js", preload: true"
      append    config/importmap.rb
  Appending: pin "@hotwired/stimulus-loading", to: "stimulus-loading.js", preload: true
      append    config/importmap.rb
  Pin all controllers
  Appending: pin_all_from "app/javascript/controllers", under: "controllers"
      append    config/importmap.rb
         run  bundle install
Bundle complete! 14 Gemfile dependencies, 82 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
         run  bundle lock --add-platform=x86_64-linux
Writing lockfile to /Users/faraaz/oss/reseed/Gemfile.lock
```

### System configuration
**Rails version**: Rails 7.1.2

**Ruby version**: ruby 3.2.2 (2023-03-30 revision e51014f9c0) [x86_64-darwin22]

**Comment by fatkodima** [comment]
This seems like a feature request, but we reserve github issues for issues in the rails framework. 
Please use the [Rails forum](https://discuss.rubyonrails.org/) for the discussion first.

My personal opinion - this is not so scary and not much of a problem, you will get used to it after the first time or so.

**Comment by zzak** [comment]
This will be addressed in 7.2 thanks to #50185 🎉
